docker/docker-compose.yml

```diff
@@ -27,14 +27,14 @@ services:
     deploy: *default-deploy
     volumes:
       - influxdb:/var/lib/influxdb/
-    healthcheck:
```

Comment:
Why no healthcheck?

Reply:
Not needed. The setup script waits until the service is ready before creating the databases. I've seen issues with healthchecks, especially in rabbitmq.

Reply:
But the healthcheck runs continually during the service, not just during startup.

Reply:
Ok, i'll add it back.

Reply:
Was it breaking something by having it there?

Reply:
Let me add it back in and test. I don't remember the exact message I was seeing before.